tests/test_agr_mvs.py

```diff
@@ -0,0 +1,152 @@
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from agr_mvs.enum_turno import TipoTurno
+from agr_mvs.empresa import Empresa
+from agr_mvs.tienda import Tienda
+from agr_mvs.Turno import Turno
+from agr_mvs.Ticket import Ticket
+from hamcrest import *
+import pytest
+from datetime import datetime
+
+empresa = Empresa("NRSUR")
+tienda = Tienda(1, "KFC", empresa)
+turno1 = Turno(1, TipoTurno.ALMUERZO, 3.0, datetime(2020, 12, 12, 00, 00))
+turno2 = Turno(2, TipoTurno.CENA, 3, datetime(2020, 12, 12, 00, 00))
+ticket = []
+ticket2 = []
+minutos = [20, 25, 30, 35, 40, 45, 50, 55]
```

Comment:
Lo pones aquí, pero no lo usas...

Reply:
los array los utilizo dentro de los bucles for y el array de minutos es necesario y lo uso para ir introduciendo tickets

Reply:
los array los utilizo dentro de los bucles for y el array de minutos es necesario y lo uso para ir introduciendo tickets